src/client/src/models/Scene.js

```diff
@@ -108,22 +108,47 @@ const Scene = {
     Scene.setScenes(list)
   },
 
+
+
+
+
+
+
   // this function will save the current scene to the localStorage
   saveStart: function() {
     const list = Scene.getScenes()
 
     // this is a check to make sure the value entered into start is a number
     const start_check = Number(Scene.currentScene.start)
```

Comment:
This needs to be a bit more verbose. Number() returns any empty string, or string filled with white space with 0 so this function has a bit of a bug in it. To fix it we first need to cleanse the start of white space using the member function `.replace(/\s/g, "")` This will strip it of white space.

Reply:
Done handled by the `start_replace`.